src/ipc/ipc4/handler.c

```diff
@@ -1066,7 +1066,8 @@ void ipc_cmd(struct ipc_cmd_hdr *_hdr)
 	if (!in)
 		return;
 
-	tr_dbg(&ipc_tr, "rx\t: %#x|%#x", in->primary.dat, in->extension.dat);
+	if (cpu_is_primary(cpu_get_id()))
+		tr_info(&ipc_tr, "ipc: new cmd %#x|%#x", in->primary.dat, in->extension.dat);
```

Comment:
The format was specifically aligned with the kernel format (for IPC4). But not that important for me.

For merging, we do need to check test results (that they don't get worse), I was worried about log overhead and thus tr_dbg() by default in original patch.

Reply:
@kv2019i well, the format isn't that important for me, I thought it would be good to have it similar to IPC3. What's the benefit of having it like the kernel? :-) But I'm fine either way. As for the overhead - to me I think this is the single most important print whenever I debug the firmware (i.e. always). I wouldn't mind dropping **all** other log entries and only enabling them on request but this one is **really** the one that allows me to synchronise firmware logs with kernel logs. And usually they aren't very frequent either - you use a handful to build the pipeline, then you start-stop - that's usually it.